include/glow/Backend/Backend.h

```diff
@@ -39,7 +39,6 @@ enum class BackendKind {
 };
 
 /// Stop gap measure while migrating from BackendKind to backend names.
-/// TODO: Remove this after migration.
```

Comment:
Is this migration no longer happening?

Reply:
No idea, but if we have BackendKind we should have a kind->name map.

Reply:
Ah I see what you mean since we're removing BackendKind (e.g. in #3087). Will rebase and update.